addons/account/models/account.py

```diff
@@ -679,7 +679,7 @@ def compute_all(self, cr, uid, ids, price_unit, currency_id=None, quantity=1.0,
     def _fix_tax_included_price(self, price, prod_taxes, line_taxes):
         """Subtract tax amount from price when corresponding "price included" taxes do not apply"""
         # FIXME get currency in param?
-        incl_tax = prod_taxes.filtered(lambda tax: tax.id not in line_taxes and tax.price_include)
+        incl_tax = prod_taxes.filtered(lambda tax: tax.id not in line_taxes.ids and tax.price_include)
```

Comment:
`incl_tax = prod_taxes.filtered(lambda tax: tax not in line_taxes and tax.price_include)`